src/vm/win32threadpool.h

```diff
@@ -372,8 +372,16 @@ class ThreadpoolMgr
 
         } counts;
 
+        // padding to ensure we get our own cache line
+        BYTE padding[64];
+
         Counts GetCleanCounts()
         {
+#ifdef _WIN64
+            // VolatileLoad x64 bit read is atomic
+            return DangerousGetDirtyCounts();
```

Comment:
On second thought, I think as you said, the main reason it was using interlocked load is to prevent torn reads on 32-bit platforms. I think what you have here is fine for now.